products/pdt/cd/base/deployment.yaml

```diff
@@ -1,32 +1,119 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: pdt-cronjob
+  namespace: pdt
+  labels:
+    "app.kubernetes.io/name": "pdt-cronjob"
+spec:
+  schedule: "0 0 * * *"
+  selector:
+    matchLabels:
+      "app.kubernetes.io/name": "pdt-cronjob"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: default
+          env:
+            - name: PROJECT_ID
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: PROJECT_ID
+            - name: DATASET_ID
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: DATASET_ID
+            - name: NETWORK
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: NETWORK
+            - name: DOWNLOAD_DIR
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: DOWNLOAD_DIR
+            - name: UNPACK_DIR
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: UNPACK_DIR
+            - name: NR_THREADS
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: NR_THREADS
+            - name: BATCH_BLOCKS
+              valueFrom:
+                configMapKeyRef:
+                  name: pdt
+                  key: BATCH_BLOCKS
+          command: ["sh", "-c"]
+          args:
+            - |
+              ./pdt --download-dir=$DOWNLOAD_DIR --unpack-dir=$UNPACK_DIR --network=$NETWORK download
+              ./pdt --download-dir=$DOWNLOAD_DIR --unpack-dir=$UNPACK_DIR bqmulti --project-id=$PROJECT_ID --dataset-id=$DATASET_ID --nr-threads=$NR_THREADS --batch-blocks=$BATCH_BLOCKS
+          containers:
+            - image: pdt
+              name: pdt-cronjob
+              volumeMounts:
+                - mountPath: /data
+                  name: data
+          volumes:
+            - name: data
+              persistentVolumeClaim:
+                claimName: pdt-pvc
```

Comment:
Only the cronjob requires access to the volumes for download and upload any gaps to BQ